catalog/core/catalog-core-downloadaction/src/main/java/org/codice/ddf/catalog/download/action/ResourceDownloadActionProvider.java

```diff
@@ -118,8 +118,8 @@ private boolean hasRemoteSiteName(Metacard metacard) {
     Optional<String> sourceId = Optional.ofNullable(metacard.getSourceId());
 
     if (sourceId.isPresent()) {
-      String sourceIdLowerCase = sourceId.get().toLowerCase();
-      return !sourceIdLowerCase.equals(getLocalSiteName());
+      String sourceIdLowerCase = sourceId.get();
```

Comment:
✏️ Not the least because this variable's name is now wrong...I'd suggest just inlining `sourceId.get()` instead of bothering with the interim variable.